project.clj

```diff
@@ -84,7 +84,7 @@
                  ["snapshots" "http://nexus.delivery.puppetlabs.net/content/repositories/snapshots/"]]
 
   :plugins [[lein-parent "0.3.1"]
-            [puppetlabs/i18n "0.4.3"]]
+            [puppetlabs/i18n "0.5.1"]]
```

Comment:
@rlinehan Are you good with this change?  Wasn't sure if you had more insight here than the rest of us do.